### PR TITLE
feat: Add more features to testing framework.

### PR DESCRIPTION
**Pull request by itsdevbear**

### CODE REVIEW

File: cosmos/testing/integration/fixture.go [hunk 1]
```diff
@@ -36,6 +41,7 @@ const defaultTimeout = 10 * time.Second
 // TestFixture is a testing fixture that can be used to test the
 // Ethereum JSON-RPC API.
 type TestFixture struct {
+	t           network.TestingT
```
Comment (transmissions12) [inline cosmos/testing/integration/fixture.go:44]: why did we add t here? maybe update the name?

**Review by transmissions12 — APPROVED** [review]
looks good, we can add more utility functions as we go

**Comment by itsdevbear** [comment]
So that we can use t.fatal, t is a very common name for this variable

**Comment by itsdevbear** [comment]
Trying to make it so that the fixture is not opinionated to Ginkgo, eventually we can make a test suite that chain unopionated as well.

**Pull request merged**